build/Makefile

```diff
@@ -41,29 +41,42 @@ SCRIPTDIR ?= $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 DSTDIR ?= $(PDO_INSTALL_ROOT)
 SRCDIR ?= $(abspath $(SCRIPTDIR)/..)
 
-KEYDIR := $(DSTDIR)/opt/pdo/keys/
-ETCDIR := $(DSTDIR)/opt/pdo/etc/
-
-KEYGEN=$(abspath $(SCRIPTDIR)/__tools__/make-keys)
-CNFGEN=$(abspath $(SCRIPTDIR)/__tools__/expand-config)
-BUILD=$(abspath $(SCRIPTDIR)/__tools__/build.sh)
-VERIFY_PRE_BUILD=$(abspath $(SCRIPTDIR)/__tools__/verify-pre-build.sh)
-VERIFY_PRE_CONF=$(abspath $(SCRIPTDIR)/__tools__/verify-pre-conf.sh)
-CLEAN=$(abspath $(SCRIPTDIR)/__tools__/clean.sh)
-TESTSCRIPT=$(abspath $(SCRIPTDIR)/__tools__/run-tests.sh)
-BENCHMARKSCRIPT=$(abspath $(SCRIPTDIR)/__tools__/run-benchmarks.sh)
+# this is an optional configuration variable to control the
+# hostname used in configuration files; we use it here to
+# customize the site.psh file
+ifndef PDO_HOSTNAME
+SERVICE_HOST = localhost
+else
+SERVICE_HOST = $(PDO_HOSTNAME)
+endif
```

Comment:
Can't we simply use `PDO_HOSTNAME` (rather than defining another variable)?

`PDO_HOSTNAME` is set in common config and checked in config.sh (which is used sort of everywhere).

Reply:
I had a related question earlier. See Mic's response: https://github.com/hyperledger-labs/private-data-objects/pull/412#discussion_r1135702513

Reply:
SERVICE_HOST should be conditional with the default as PDO_HOSTNAME (e.g. because clients may want to configure to use services running on a different server).  i'll fix this up (not PDO_HOSTNAME should always be defined to "something" so this doesn't really make sense anyway.